packages/lesmis/src/components/Breadcrumbs.js

```diff
@@ -36,38 +36,30 @@ const Link = props => <MuiLink color="inherit" {...props} component={RouterLink}
 
 const Loader = () => (
   <Skeleton animation="wave">
-    <MuiLink>Loading</MuiLink>
+    <MuiLink>breadcrumbsLoading</MuiLink>
```

Comment:
this string just mocks the length of the skeleton loader. The text doesn't actually show.